packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -23,6 +23,19 @@ export enum UsageDataLevel {
   on = "on",
 }
 
+/**
+ * Defines an error that is expected to happen given some situation
+ * @member err Message to be logged in the error
+ */
+export class ExpectedError extends Error {
+  constructor(err: string) {
+      super(err);
+
+      // Set the prototype explicitly.
+      Object.setPrototypeOf(this, ExpectedError.prototype);
```

Comment:
I'm not familiar with this but I wouldn't think this is the way to do this in TypeScript.

Reply:
The Error() constructor type definition is `Error(message: string | undefined)` so the constructor here should match that.